### PR TITLE
Fix #671 and fix #672

### LINKED ISSUE #671: §2.2: IA Sequence definition
|Version|Source|Clause(s)|
|:---:|:---:|:---:|
|AOM Working Group Draft, 17 July 2023|Apple|§2.2|

#### Comment:
<blockquote>
- IA Sequence is defined in § 2.3.1 IA Sequence.
</blockquote>

The actual definition spreads across multiple sections.

-----
#### Suggestion:
It would be beneficial to point out all sections which one needs to read to understand what IA Sequence is. The definition itself can be improved as will be discussed in another issue.

### LINKED ISSUE #672: §2.3: IA Sequence definition
|Version|Source|Clause(s)|
|:---:|:---:|:---:|
|AOM Working Group Draft, 17 July 2023|Apple|§2.3|

#### Comment:
This clause could be improved. When does the IA Sequence start and when does it end? Maybe it would be better to say that it is a sequence of immersive audio OBUs starting with the sequence header OBU etc.

-----
#### Suggestion:
Improve the definition of the IA Sequence as it is a very important construct in the specification. Current definition requires you to dig into multiple clauses to understand what it is.

### PR DESCRIPTION
**Pull request by sunghee-hwang**
***
<a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/pull/707.html" title="Last updated on Aug 21, 2023, 12:54 AM UTC (4883dbd)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/707/5c6c3b5...4883dbd.html" title="Last updated on Aug 21, 2023, 12:54 AM UTC (4883dbd)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -347,6 +347,8 @@ An [=IA Sequence=] is a bitstream to represent [=Immersive Audio=] contents and
 
 The metadata in the [=Descriptors=] and [=IA Data=] are packetized into individual Open Bitstream Units (OBU)s. The term Open Bitstream Unit (OBU) is the concrete, physical unit used to represent the components in the model.
```
Comment (sunghee-hwang) [inline index.bs:348]:
```suggestion
The metadata in the [=Descriptors=] and [=IA Data=] are packetized into individual Open Bitstream Units (OBU)s. The term Open Bitstream Unit (OBU) is the concrete, physical unit used to represent the components in the model. In this version of the specification, IA OBU is denoted as OBU for convenience.
```

File: index.bs [hunk 2]
```diff
@@ -341,11 +341,13 @@ For a given input 3D audio,
 
 ## Bitstream Structure ## {#bitstream}
 
-### IA Sequence ### {#iasequence}
+### Overview ### {#iasequence}
```
Comment (felicialim) [inline index.bs:344]:
```suggestion
### Overview ### {#overview}
```

**Review by felicialim — COMMENTED** [review]
> - [=IA Sequence=] is defined in [[#iasequence]].

This should point to [[#standalone-ia-sequence]] instead.

File: index.bs [hunk 3]
```diff
@@ -341,11 +341,13 @@ For a given input 3D audio,
...
 
 An [=IA Sequence=] is a bitstream to represent [=Immersive Audio=] contents and consists of [=Descriptors=] and [=IA Data=].
 
-The metadata in the [=Descriptors=] and [=IA Data=] are packetized into individual Open Bitstream Units (OBU)s. The term Open Bitstream Unit (OBU) is the concrete, physical unit used to represent the components in the model.
+The metadata in the [=Descriptors=] and [=IA Data=] are packetized into individual Open Bitstream Units (OBU)s. The term Open Bitstream Unit (OBU) is the concrete, physical unit used to represent the components in the model. In this version of the specification, IA OBU is denoted as OBU for convenience.
```
Comment (felicialim) [inline index.bs:348]: We don't define or use IA OBU at this point yet so I suggest the following:

```suggestion
The metadata in the [=Descriptors=] and [=IA Data=] are packetized into individual Open Bitstream Units (OBU)s. The term Open Bitstream Unit (OBU) is the concrete, physical unit used to represent the components in the model. In this version of the specification, the term IA OBU can be used interchangeably with OBU.
```

**Review by felicialim — APPROVED** [review]

**Pull request merged**